tests/wells.py

```diff
@@ -93,3 +93,8 @@ def testCompletions(self):
         c0,c1 = w0.completions()
         self.assertEqual((6,6,2), (c0.getI(), c0.getJ(), c0.getK()))
         self.assertEqual((6,6,3), (c1.getI(), c1.getJ(), c1.getK()))
+
+
+if __name__ == '__main__':
+    unittest.main()
+    
```

Comment:
Please run whitespace-fixup on all your files.  They should all end with one newline character and have not trailing space.

(Configure your editor to do this for you.)